app/Views/admin/certificates/Certificates.php

```diff
@@ -0,0 +1,108 @@
+<link href="/assets/common/css/sweetAlert.min.css" rel="stylesheet">
+<?= $this->endSection() ?>
+
+<?= $this->section('content') ?>
+<div class="page-content">
+    <div class="container-fluid">
+
+        <?= view('partials/page-title', ['title' => 'Certificados']); ?>
+
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="card">
+                    <div class="card-header d-flex justify-content-between align-items-center flex-wrap">
+                        <h5 class="card-title mb-2 mb-md-0">Tabla de Certificados</h5>
+                        <div class="mt-2 mt-md-0 float-md-right">
+                            <a href="/admin/certificados/crear" class="btn btn-success btn-label waves-effect waves-light rounded-pill">
+                                <i class="ri-add-circle-fill label-icon align-middle rounded-pill fs-16 me-2"></i>Agregar Certificado
+                            </a>
+                        </div>
+                    </div>
+
+                    <?php
+                        if (session()->has('response')) {
+                            $response = session()->get('response');
+                            ?>
+                        <div id="alertElement" data-response="<?= htmlspecialchars(json_encode($response)) ?>"></div>
+                    <?php } ?>
+
+                    <!-- card-header -->
+                    <div class="card-body">
+                        <table id="certificates-table" class="table nowrap dt-responsive align-middle table-hover table-bordered" style="width:100%">
+                            <thead>
+                                <tr>
+                                    <th>ID</th>
+                                    <th>Nombre</th>
+                                    <th>Previsualización de la imagen</th>
+                                    <th>Acciones</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <?php foreach ($certificates as $certificate) { ?>
+                                    <tr>
+                                        <td><?= $certificate['certificateId']; ?></td>
+                                        <td>
+                                            <div class="d-flex align-items-center fw-medium">
+                                                <p class="text-wrap"><?= $certificate['certificatefileName'] ?></p>
+                                            </div>
+                                        </td>
+                                        <td>
+                                            <div class="d-flex justify-content-center align-items-center">
+                                                <img src="<?= $certificate['fileRoute']; ?>" alt="Imagen del certificado " . <?= $certificate['certificateId']; ?> width="200" height="100">
```

Comment:
Add the "mw-100h-auto" classes
```suggestion
                                                <img class="mw-100 h-auto" src="<?= $certificate['fileRoute']; ?>" alt="Imagen del certificado " . <?= $certificate['certificateId']; ?> width="200" height="100">
```